src/widget/analog-scale/analog-scalepicker.scss

```diff
@@ -210,3 +210,7 @@ $q-pad-right: 0px !default;
         @include display-flex;
     }
 }
+
+.pages.or [role="page"]:not(.current) {
```

Comment:
This rule is not specific to analog-scale widgets. I think the fix is good, but it should be made around here I think: https://github.com/enketo/enketo-core/blob/master/src/sass/core/_pages.scss#L17

Alternatively, perhaps instead of this change, perhaps a different kind of change in analog-scalepicker.scss could be made to make sure the current pages.scss rule 'wins' (I'm not sure how).

Reply:
Got it. I think I know where the specific changes should be made.